source/dagonBackend.d

```diff
@@ -1403,7 +1404,8 @@ final class SacScene: Scene{
 			if(eventManager.keyPressed[KEY_I]) speed = 10.0f;
 			if(eventManager.keyPressed[KEY_O]) speed = 100.0f;
 			if(eventManager.keyPressed[KEY_P]) speed = 1000.0f;
-			if(dir.lengthsqr>0.001f^^2) camera.position += dir.normalized * speed * (dt.total!"hnsecs"*1e-7);
+			if(eventManager.keyPressed[KEY_LSHIFT]) factor *= 2;
```

Comment:
You can declare `factor` in front of this line to reduce its scope.

Reply:
Should be `factor *= 2.0f`. I adhere to this convention to make it a bit more clear where the floating-point operations are.

Reply:
alright, fair point.